sycl/cmake/modules/FetchUnifiedRuntime.cmake

```diff
@@ -116,14 +116,15 @@ if(SYCL_UR_USE_FETCH_CONTENT)
       CACHE PATH "Path to external '${name}' adapter source dir" FORCE)
   endfunction()
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit 7a2caca559da81620c7430521873045bf42eafc1
-  # Merge: 1d1808a4 96f66e0f
-  # Author: Piotr Balcer <piotr.balcer@intel.com>
-  # Date:   Tue Sep 24 13:00:46 2024 +0200
-  #     Merge pull request #2117 from pbalcer/fix-filter-out-same-cmdlists
-  #     Fix urEnqueueEventsWaitWithBarrier when used with interop events
-  set(UNIFIED_RUNTIME_TAG 7a2caca559da81620c7430521873045bf42eafc1)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/frasercrmck/unified-runtime.git")
+    
+  # commit cabf128094eff9ff7b79bdff559640a8a111f0c3
+  # Merge: a96fcbc5 15bca3b6
+  # Author: Omar Ahmed <omar.ahmed@codeplay.com>
+  # Date:   Mon Aug 19 16:20:45 2024 +0100
+  #     Merge pull request #1984 from rafbiels/rafbiels/cuda-stream-race-cond
+  #     Fix race condition in CUDA stream creation
+  set(UNIFIED_RUNTIME_TAG 6eb5208bad81457d367dbb5b51ad266cda7cf7c1)
```

Comment:
https://github.com/oneapi-src/unified-runtime/pull/1996 is merged, please update

Reply:
done, thanks!